crates/configure-holochain/src/hpos_holochain_api.rs

```diff
@@ -10,7 +11,7 @@ struct HostingCriteria {
 }
 
 pub async fn get_jurisdiction() -> Result<String> {
-    let output: Output = Command::new("hpos-holochain-client")
+    let output: Output = Command::new("/run/current-system/sw/bin/hpos-holochain-client")
```

Comment:
We should update holo-nixpkgs to make the hpos-holochain-client avaliable

Reply:
I actually saw a [different pattern](https://github.com/Holo-Host/holo-auto-installer/blob/99f2849811cbf3a0c2cc7ffc16afe1a825d89731/src/install_app.rs#L104-L113) that doesn't call the hpos-holochain-client at all, but instead just makes an http post call to hpos-holochain-api directly. What do you think about replacing this pattern (here and in autoinstaller) with that pattern?